### DISCUSSION TITLE
Propose pagefind as alternative to FlexSearch

### DISCUSSION THREAD

**Discussion by nickwe3** [colinwilson/lotusdocs#144] [Feature Requests]
Hello,

Would you mind considering pagefind as alternative to FlexSearch ?

It's an opensource project and since they hit their first major version 1.0 this September 2023, they're the new kids in the place:

* https://pagefind.app/

Results are similar to DocSearch/Algolia.

Starlight theme from Astro is already using it.

Thanks and Regards,

Nicolas.

**Comment by colinwilson** [comment]
Looks interesting. I don't see why it can't be added as another option. Will look into it. Thanks. 👍
